sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -586,9 +647,99 @@
+				"template": {
+					"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+					"contentVersion": "1.0.0.1",
+					"resources": [{
+						"apiVersion": "2022-01-01-preview",
+						"name": "[parameters('workspaceName')]",
+						"location": "[parameters('location')]",
+						"kind": "featurestore",
+						"type": "Microsoft.MachineLearningServices/workspaces",
+						"identity": {
+							"type": "SystemAssigned,UserAssigned",
+							"userAssignedIdentities": {
+							    "[parameters('materialization_identity_resource_id')]": {}
+							}
+						},
+						"properties": {
+                            "friendlyName": "[parameters('friendlyName')]",
+                            "description": "[parameters('description')]",
+                            "storageAccount": "[variables('storageAccount')]",
+                            "keyVault": "[variables('keyVault')]",
+                            "applicationInsights": "[variables('applicationInsights')]",
+                            "containerRegistry": "[if(not(equals(parameters('containerRegistryOption'), 'none')), variables('containerRegistry'), json('null'))]",
+                            "hbiWorkspace": "[parameters('confidential_data')]",
+                            "imageBuildCompute": "[parameters('imageBuildCompute')]",
+                            "publicNetworkAccess": "[parameters('publicNetworkAccess')]",
+                            "softDeleteEnabled": "[parameters('soft_delete_enabled')]",
+                            "allowRecoverSoftDeletedWorkspace": "[parameters('allow_recover_softdeleted_workspace')]",
+                            "encryption": {
+                                "status": "[parameters('encryption_status')]",
+                                "keyVaultProperties": {
+                                    "keyVaultArmId": "[parameters('cmk_keyvault')]",
+                                    "keyIdentifier": "[parameters('resource_cmk_uri')]"
+                                },
+                                "cosmosDbArmId": "[parameters('encryption_cosmosdb_resourceid')]",
+                                "storageAccountArmId": "[parameters('encryption_storage_resourceid')]",
+                                "SearchAccountArmId": "[parameters('encryption_search_resourceid')]"
+                            },
+                            "primaryUserAssignedIdentity": "[parameters('primaryUserAssignedIdentity')]",
+                            "featureStoreSettings": {
+                                "computeruntime": {
+                                    "SparkRuntimeVersion": "[parameters('spark_runtime_version')]"
+                                },
+                                "offlinestoreconnectionname": "[parameters('offline_store_connection_name')]",
+                                "onlinestoreconnectionname": "[parameters('online_store_connection_name')]"
+                            }
+						}
```

Comment:
Indent this part properly

Reply:
![image](https://user-images.githubusercontent.com/110191178/222005849-37578062-7ff9-4fc2-a26b-a63f245968a9.png)
Its intended in code. Not sure why it is looking like this here